From b2aac6115763bdfddb5adf45e6e3d4e193da8589 Mon Sep 17 00:00:00 2001
From: Jason Gerecke <jason.gerecke@wacom.com>
Date: Wed, 10 Feb 2021 12:44:24 -0800
Subject: [PATCH libwacom 06/10] data: Add ISDv4 5216 (Lenovo IdeaPad Flex 5
 14IIL05) (#350)

Link: https://github.com/linuxwacom/wacom-hid-descriptors/issues/103
Signed-off-by: Jason Gerecke <jason.gerecke@wacom.com>
(cherry picked from commit 5e2c94f9ba409db83acb5fe4e7d757c9ea4cf888)
---
 data/isdv4-5216.tablet | 22 ++++++++++++++++++++++
 1 file changed, 22 insertions(+)
 create mode 100644 data/isdv4-5216.tablet

diff --git a/data/isdv4-5216.tablet b/data/isdv4-5216.tablet
new file mode 100644
index 0000000..5863d98
--- /dev/null
+++ b/data/isdv4-5216.tablet
@@ -0,0 +1,22 @@
+# Lenovo IdeaPad Flex 5 14IIL05
+# Sensor Type: AES
+# Features: Touch (Integrated), Tilt
+# HW Resolution: 30937 x 17402 (2540 x 2540 lpi)
+#
+# Autogenerated from sysinfo.n0ilTsFMOk
+# https://github.com/linuxwacom/wacom-hid-descriptors/issues/103#issuecomment-768913777
+
+[Device]
+Name=ISDv4 5216
+ModelName=
+DeviceMatch=i2c:056a:5216
+Class=ISDV4
+Width=12
+Height=7
+IntegratedIn=Display;System
+Styli=@isdv4-aes;
+
+[Features]
+Stylus=true
+Touch=true
+Buttons=0
-- 
2.31.1

